### DISCUSSION TITLE
Hola explorers, busco equipo

### DISCUSSION THREAD

**Discussion by geraodst117** [Launch-X-Latam/MisionFrontEnd#570] [Team Posts]
Me gustaría formar un equipo para el hackaton y también que estudiemos mucho y hagamos proyectos para practicar

**Comment by JoselinMiel** [comment]
Yo estoy interesado este es correo ***@***.*** para crear grupo

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by JuanRodriguezDV** [comment]
lx08166@innovaccion.mx este es mi correo, tambien quiero participar

**Comment by Ayuya2001** [comment]
Hola, ya te create tu grupo?, O les gustaría unirse al mío ahora somos 2 y
justo los 3 que respondieron aquí estaríamos perfecto

**Comment by Dhinestroza05** [comment]
Buenas tardes yo quiero ser parte de este grupo

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by LuisSerranoGtz** [comment]
lx02079@innovaccion.mx 
todavia tienen lugar?

**Comment by Crosiux** [comment]
Quisiera unirme con ustedes
